protocol/near/wrapper/src/__tests__/e2e/format.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { NearPluginConfig } from "../../../../plugin-js"; //TODO change to appropriate package
+import * as testUtils from "../testUtils";
+
+import { PolywrapClient } from "@polywrap/client-js";
+import { BigInt } from "@polywrap/wasm-as";
+import * as nearApi from "near-api-js";
+import {
+  initTestEnvironment,
+  stopTestEnvironment,
+  providers,
+  ensAddresses,
+} from "@polywrap/test-env-js";
+
+jest.setTimeout(360000);
+
+describe("Amount format", () => {
+  let client: PolywrapClient;
+  let apiUri: string;
+  let nearConfig: NearPluginConfig;
+
+  beforeAll(async () => {
+    // set up test env and deploy api
+    await initTestEnvironment();
+
+    const absPath = __dirname + "/../../../build";
+    apiUri = `fs/${absPath}`;
+    const polywrapConfig = testUtils.getPlugins(
+      providers.ethereum,
+      ensAddresses.ensAddress,
+      providers.ipfs,
+      nearConfig
+    );
+    client = new PolywrapClient(polywrapConfig);
+  });
+
+  afterAll(async () => {
+    await stopTestEnvironment();
+  });
+
+  it.each(testUtils.valuesToFormat)("Format Near amount", async (amount) => {
+    const result = await client.invoke<string>({
+      uri: apiUri,
+      method: "formatNearAmount",
+      args: {
+        amount,
+      },
+    });
+    expect(result.error).toBeFalsy();
+    expect(result.data).toBeTruthy();
+
+    expect(result.data).toBeTruthy();
+    expect(result.data).toEqual(
+      nearApi.utils.format.formatNearAmount(amount, 24)
+    );
+  });
+
+  it.each(testUtils.valuesToParse)("Parse near amount", async (amount) => {
```

Comment:
Shouldn't `test` keyword suite here better? 
- You could also consider adding `should` and reword it to make it readable sentence.

Don't want to get too pedantic but I believe we should be consistent and use both keywords in the recommended ways.